include/SH3/arc/vfile.hpp

```diff
@@ -53,9 +49,11 @@ struct sh3_arc_vfile final
      *  @param mft      The @ref sh3_arc Master File Table, arc.arc.
      *  @param fname    The name of the file we want to open.
      *
+     *  @note If the file is already open, this function does nothing but returns @ref open.
```

Comment:
Very true, I was considering making this private, but I thought it might be better to give more control. Making it private makes it much safer though.